TheBereftSouls.csproj

```diff
@@ -7,10 +7,18 @@
 	<PropertyGroup>
 		
 	</PropertyGroup>
-
-	<!-- References -->
 	<ItemGroup>
-		
+	  <Reference Include="CalamityMod">
+	    <HintPath>..\Llib\CalamityMod.dll</HintPath>
+	  </Reference>
+	  <Reference Include="SOTS">
+	    <HintPath>..\Llib\SOTS.dll</HintPath>
+	  </Reference>
+	  <Reference Include="ThoriumMod">
+	    <HintPath>..\Llib\ThoriumMod.dll</HintPath>

```

Comment:
I don't have access to these folders outside of the repo so its impossible for me to review any other code due to missing assembly references everywhere. Try implementing these path hints in a folder inside the repo itself like this:

- [JavidPack/ItemChecklist/lib/](https://github.com/JavidPack/ItemChecklist/tree/ccfcfc1e881aa099ad587dcd3bae7e43a5f48254/lib)
- [JavidPack/ItemChecklist/ItemChecklist.csproj#L9](https://github.com/JavidPack/ItemChecklist/blob/ccfcfc1e881aa099ad587dcd3bae7e43a5f48254/ItemChecklist.csproj#L9)